test/verify/check-active-pages

```diff
@@ -32,23 +32,23 @@ class TestActivePages(MachineCase):
         b.wait_present("#server")
 
         def showPagesAssertCount(count):
-            # check the pages and make sure there is only one page loaded
             b.switch_to_top()
             b.mouse("#navbar-dropdown", "click", altKey=True)
             b.click("#active-pages")
             b.wait_present("#active-pages-dialog")
             b.call_js_func("ph_count_check", "#active-pages-dialog tbody", count)
 
-        showPagesAssertCount(1)
+        # Initially we have /system and /playground/preloaded
+        showPagesAssertCount(2)
```

Comment:
also needs to be behave differently on [distropkg](https://209.132.184.41:8493/logs/pull-12534-20190814-064040-567c60d4-cockpit-project-cockpit-rhel-8-0-distropkg/log.html#233)

Reply:
Done, but not actually tested on rhel-8-0-distropkg.